ntoskrnl/se/debug.c

```diff
@@ -69,7 +69,7 @@ SepDumpAceFlags(
 #define ACE_FLAG_PRINT(x)  \
     if (AceFlags & x)      \
     {                      \
-        DbgPrint(#x "\n"); \
+        DPRINT(#x "\n"); \
```

Comment:
Using DPRINT(1) would make the dumps (when debugging is enabled at compile-time) rather long with the useless file-name+line numbers. Wouldn't it be preferrable to have something like:
```c
#ifdef NDEBUG
#define ACE_FLAG_PRINT(x)
#else
#define ACE_FLAG_PRINT(x) { /* its actual definition */ }
#endif /* NDEBUG */
```
and also have those SepDump*** functions defined only when `NDEBUG` is not defined? (and otherwise defined to nothing if `NDEBUG` is defined?)
This may help also reducing the size of the PR diff.

Reply:
My aim was not to create the best debugging system ever,
but if you want to implement something like this feel free to do so.